Library/Homebrew/formula_auditor.rb

```diff
@@ -622,6 +622,26 @@ def audit_eol
       problem "Product is EOL since #{metadata["eol"]}, #{see_url}" if Date.parse(metadata["eol"]) <= Date.today
     end
 
+    def audit_wayback_url
+      return unless @core_tap
+      return if formula.deprecated? || formula.disabled?
+
+      regex = %r{^https?://web\.archive\.org}
+      if regex.match?(formula.stable.url)
+        problem "Formula with a Wayback Machine `url` should be deprecated with `:repo_removed`"
+      end
+
+      if regex.match?(formula.homepage)
+        problem "Formula with a Wayback Machine `homepage` should find an alternative `homepage`"
+      end
+
+      return unless formula.head
+
+      return unless regex.match?(formula.head.url)
+
+      problem "Formula with a Wayback `head` should not have a `head` definition"
```

Comment:
```suggestion
      problem "Formula should not have a Internet Archive Wayback Machine `head` URL"
```

if I've understood what this does correctly.

Reply:
I think if your `head` source is no longer available it should just be removed.

Reply:
Yours sounds to me like "go find a new source" while mine sounds like "remove the head block/method"

Reply:
```suggestion
      problem "Remove Internet Archive Wayback Machine `head` URL"
```

how about this?